opennms-container/horizon/container-fs/confd/templates/slack.properties.tpl

```diff
@@ -2,17 +2,22 @@
 # DON'T EDIT THIS FILE :: GENERATED WITH CONFD
 #
 
-export OPENNMS_NOTIFD_SLACK_CHANNEL
-export OPENNMS_NOTIFD_SLACK_USERNAME
-export OPENNMS_NOTIFD_SLACK_ICONEMOJI
-export OPENNMS_NOTIFD_SLACK_ICONURL
-export OPENNMS_NOTIFD_SLACK_USESYSTEMPROXY
+{{ if getenv "OPENNMS_NOTIFD_SLACK_CHANNEL" }}
+
+org.opennms.netmgt.notifd.slack.channel={{getenv "OPENNMS_NOTIFD_SLACK_CHANNEL" "Webhook"}}
+org.opennms.netmgt.notifd.slack.username={{getenv "OPENNMS_NOTIFD_SLACK_USERNAME" "none"}}
+org.opennms.netmgt.notifd.slack.iconEmoji={{getenv "OPENNMS_NOTIFD_SLACK_ICONEMOJI" ""}}
+org.opennms.netmgt.notifd.slack.iconURL={{getenv "OPENNMS_NOTIFD_SLACK_ICONURL" ""}}
+org.opennms.netmgt.notifd.slack.useSystemProxy={{getenv "OPENNMS_NOTIFD_SLACK_USESYSTEMPROXY" "true"}}
+
+{{ else }}
```

Comment:
You can get rid of this along with the conditional. When confd is invoked you can tell it to source keys from the environment rather than yaml or another key source. We don't want to be explicitly checking for the env vars in the template.

Once that is done I'm good with this PR. I'm not familiar with what confd key source horizon is using so I don't need to see the env test work if we aren't using it, Ronny could provide guidance there perhaps.